src/webots/nodes/WbTrack.cpp

```diff
@@ -908,8 +885,7 @@ void WbTrack::exportAnimatedGeometriesMesh(WbVrmlWriter &writer) const {
 
   if (writer.isX3d()) {
     writer << "<Transform ";
-    writer << "translation='" << position << "' ";
-    writer << "rotation='" << rotation << "'>";
+    writer << " type='animatedgeometry'>";
```

Comment:
Also I am not sure why we write 'animatedgeometry' here whereas I would have written 'track', but track seems to be used elsewhere as well. Not sure what is the difference...

Reply:
`type=track` is for the whole node `track` whereas `type=animatedgeometry` is for the track belt element (that is called `animatedgeometry` in Webots.

Reply:
As discussed, I would suggest to use "role" instead of "type" for this kind of information (and to update the ImageTexture code accordingly to be consistent, and maybe others). However, this could be done in a separate PR.